### DISCUSSION TITLE
Correct workflow for importing new products in a CLI script?

### DISCUSSION THREAD

**Discussion by itsnewtjam** [vendurehq/vendure#2319] [Q&A]
I've been poking around trying to setup a CLI script that we could use to periodically import a CSV of product data (updates to existing products as well as new products). I thought I was on the right track using the Populator and Importer classes but I'm not having much luck. The script runs okay, but I don't see anything reflected in the Admin UI.

I'm sure I'm just overlooking something obvious, but if anyone could point me in the right direction I'd really appreciate it!

**Comment by michaelbromley** [comment]
Hi!

Do you see anything in the database? Also, the populator is only intended for new products - it will not update existing products but will create a new one instead.

And also, you are welcome to join our Discord community: https://vendure.io/community - you will get help much faster generally.
